base/intset.jl

```diff
@@ -239,3 +245,8 @@ function hash(s::IntSet, h::UInt)
     end
     h
 end
+
+minimum(s::IntSet) = first(s)
+maximum(s::IntSet) = last(s)
+extrema(s::IntSet) = (first(s), last(s))
+issorted(s::IntSet) = true
```

Comment:
> Is the order a set is iterated in even defined

I guess yes for `Set`, but for `IntSet` it's always in sorted order, and I believe it will stay this way. I don't know whether it's documented though. This added method here is only an optimization.